### DISCUSSION TITLE
How to use this function? I can't find it on the page

### DISCUSSION THREAD

**Discussion by 440981** [stashapp/stash#3104] [Q&A] [answered]
How to use this function? I can't find it on the page
![image](https://user-images.githubusercontent.com/51017558/200714480-91a393c8-667a-4a03-88db-a108d0d0d0c1.png)
![image](https://user-images.githubusercontent.com/51017558/200714531-cdab989a-8015-47e8-9447-516336f7260f.png)

**Comment by WithoutPants** [comment, marked as answer]
Scraping is used to extract metadata about an existing scene. You have to have a scene present to scrape its metadata.

**Reply by 440981** [reply]
So how do I do this because I've already added it
![image](https://user-images.githubusercontent.com/51017558/200732139-73bbc9d3-c7a2-46ef-8e1d-708b34b2bf19.png)

**Reply by WithoutPants** [reply]
The scenes - video files - need to be scanned in, using Tasks -> Scan. When you have scenes scanned in, you can see them listed in the Scenes page, then when you click on a scene you can use scrapers to scrape the metadata. Please see the Manual for basic operation.

**Reply by 440981** [reply]
I'm sorry I still don't understand how this scene is created to scan
![image](https://user-images.githubusercontent.com/51017558/200740443-f5008c41-e761-48d7-974f-d1465000d127.png)

**Reply by DogmaDragon** [reply]
Go to Settings > Library > Add Directory > and select the location where your files are stored. 
Then go to Tasks > Scan.

**Reply by 440981** [reply]
> 转到设置 > 库 > 添加目录 > 并选择存储文件的位置。 然后转到任务 > 扫描。
I hit scan but nothing changed
![image](https://user-images.githubusercontent.com/51017558/200741080-c2f602da-0317-44a9-bc2a-1144009859cb.png)
![image](https://user-images.githubusercontent.com/51017558/200741122-8c6e86e6-d014-46da-b917-4fb49f5071b1.png)

**Reply by 440981** [reply]
Is there a detailed graphic tutorial? The tutorial in the wiki is very vague
